src/web/plugin-admin.jsp

```diff
@@ -533,7 +540,7 @@ tr.lowerhalf > td:last-child {
         <h3><fmt:message key="plugin.admin.upload_plugin" /></h3>
         <p><fmt:message key="plugin.admin.upload_plugin.info" /></p>
         <form action="plugin-admin.jsp?uploadplugin&amp;csrf=${csrf}" enctype="multipart/form-data" method="post">
-            <input type="file" name="uploadfile" />
+            <input type="file" name="uploadfile" accept="application/java-archive"/>
```

Comment:
Does this pick up `.jar` as well as `.war` in the relevant browsers? I'm worried that we're excluding the latter.